src/db/clients/postgresql.js

```diff
@@ -23,31 +23,33 @@ export default function(server, database) {
     const client = new Client(dbConfig);
 
     debug('connecting');
-    client.connect(err => {
+    client.connect(async err => {
       if (err) {
         client.end();
         return reject(err);
       }
 
       debug('connected');
+      const defaultSchema = await getSchema(client);
+
       resolve({
         wrapIdentifier,
         disconnect: () => disconnect(client),
-        listTables: () => listTables(client),
-        listViews: () => listViews(client),
-        listRoutines: () => listRoutines(client),
-        listTableColumns: (db, table) => listTableColumns(client, db, table),
-        listTableTriggers: (table) => listTableTriggers(client, table),
+        listTables: (_, schema = defaultSchema) => listTables(client, schema),
```

Comment:
Seems setting the `defaultSchema` in the function definition instead of here would be a better choice.